src/main/scala/cromwell/binding/types/WdlObjectType.scala

```diff
@@ -2,4 +2,6 @@ package cromwell.binding.types
 
 case object WdlObjectType extends WdlType {
   override def toWdlString: String = "Object"
+
+  override protected def coercion = ???
```

Comment:
I guess this is fine for this PR, though I suppose this could accept a `Map[String, Any]`?